packages/shared/src/components/squads/Details.tsx

```diff
@@ -170,6 +182,30 @@ export function SquadDetails({
               }}
             />
           )}
+          {createMode && (
+            <div className="flex flex-col justify-start">
+              <h4 className="mb-2 font-bold typo-headline text-theme-label-primary">
+                Post permissions
+              </h4>
+              <p className="mb-4 text-theme-label-tertiary typo-callout">
+                Choose who is allowed to post new content in this Squad.
+              </p>
+              <RadioItem
```

Comment:
Probably favor using `<Radio>` component, this wraps the items and you can simply pass an array of options to it.